spec/text-document-spec.coffee

```diff
@@ -2,28 +2,51 @@ Point = require "../src/point"
 TextDocument = require "../src/text-document"
 
 describe "TextDocument", ->
-  document = null
-
-  beforeEach ->
-    document = new TextDocument
+  [document, layer] = []
 
   describe "::buildDisplayLayer()", ->
     describe "::slice(start, end)", ->
       it "returns the content between the start and end points", ->
+        document = new TextDocument(softWrapColumn: 10, tabLength: 4)
```

Comment:
What do you think about moving the display layer construction in `TextDocument` constructor?